File: src/test/resources/AnnotateClassMethodsIntention/guava/before.template.after.java
import com.google.common.annotations.VisibleForTesting;

class Foo {

    @VisibleForTesting
    private void annotatedFoo() {}

    public void publicFoo() {}

    @VisibleForTesting
    public void foo() {}

    @VisibleForTesting
    public void bar() {}

    @VisibleForTesting
    public void baz() {}
}

File: src/test/resources/AnnotateKtClassOrObjectMethodsIntention/guava/before.template.after.kt
import com.google.common.annotations.VisibleForTesting

class Foo {

    @VisibleForTesting
    private fun annotatedFoo() {}

    fun publicFoo() {}

    @VisibleForTesting
    fun foo() {}

    @VisibleForTesting
    fun bar() {}

    @VisibleForTesting
    fun baz() {}
}